tests/__snapshots__/app.test.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
When running tests, got an error that snapshots were obsolete. Doesn't seem to affect core functionality of your code but if you want to use snapshot tests, I'd recommend looking into how to keep your snapshots up to date.